deployment/load-nci-dump.bash

```diff
@@ -94,38 +97,47 @@ psql ${psql_args} "${dbname}" -X -c 'copy (select name from agdc.dataset_type or
 echo "
 [datacube]
 db_database: ${dbname}
+db_hostname: db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com
```

Comment:
DB config should be passed as a variable, not hard coded

Reply:
Updated to pass db config as variables